harvester/config.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+
+import sentry_sdk
+
+# NOTE: if creating Config class, include utils.in_container()
```

Comment:
My preference would be to leave as-is for the moment.  It would be a pretty straight-forward update later, but this app hardly touches the Config, so tempted to just leave it as-is, and could update it with other apps as we finalize on the `Config` class approach.

Reply:
Just curious, that seems fine!